docs/source/user_guide/users.ipynb

```diff
@@ -0,0 +1,1591 @@
+{
```

Comment:
<blockquote>cuspatial uses <code>GeoArrow</code> buffers, a GPU-accelerated buffer type format that makes the   &nbsp;performance you get with cuspatial possible.</blockquote><p><code>GeoArrow</code> itself isn't GPU-accelerated, but is a format well suited for massively parallel programming model that works very well with GPU. </p><p> </p><blockquote>GeoArrow uses [PyArrow]( &nbsp;<a href="https://arrow.apache.org/docs/python/index.html" target="_blank">https://arrow.apache.org/d</a><a href=" &nbsp;https://arrow.apache.org/docs/python/index.html       " target="_blank">PyArrow</a></blockquote><p>GeoArrow extends <a href="link" target="_blank">Arrow</a> specification and introduces several new types suited for geometry application. Cuspatial implements GeoArrow by extending types in PyArrow and handles format conversion with existing PyArrow constructors.</p><p> </p><blockquote> Using an Arrow [DenseArray]( &nbsp;<a href="https://arrow.apache.org/docs/python/data.html#union-arrays" target="_blank">https://arrow.apache.org/docs/python/data.html#union-arrays</a>), &nbsp;GeoArrow stores heterogeneous   &nbsp;orderings of Features, similar to the [GeoSeries]( &nbsp;<a href="https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoSeries.html" target="_blank">https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoSeries.html</a>     ) used &nbsp;in [GeoPandas]( &nbsp;<a href="https://geopandas.org/en/stable/index.html" target="_blank">https://geopandas.org/en/stable/index.html</a>).</blockquote><p>What's similar? GeoPandas' geoseries also use <code>DenseArray</code> or <code>GeoPandas</code>  also supports heterogeneous orderings of features? Also, perhaps you mean heterogeneous type of column elements? "Heterogenous orderings of features" sounds like different order of features, which is supported even in usual series right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/666/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Thanks for the comments, I've revised the paragraph according to your suggestions. WRT your last paragraph, I think that "Feature" refers specifically to the spatial concept of a geometry feature and it's metadata. I've revised that section as well, is it more clear?

Reply:
<p>Hmm, I still think the use of <code>ordering</code> seems a bit strange. I believe you mean union column supports "heterogeneous type of features". If so I think just say <code>type</code> is better.</p><p> </p><blockquote> be loaded and transformed in a method similar to the geoseries...</blockquote><p> </p><p>be loaded and transformed with methods similar to those in <code>GeoPandas.GeoSeries</code>  </p>

Reply:
Done.